icon/iiss/icstate/networkvalue.go

```diff
@@ -157,6 +158,17 @@ func (s *State) GetUnbondingPeriod() int64 {
 	return getValue(s.store, VarUnbondingPeriod).Int64()
 }
 
+func (s *State) SetUnstakeSlotMax(value int64) error {
+	if value <= 0 {
+		return errors.IllegalArgumentError.New("unstakeSlotMax must be positive number")
+	}
+	return setValue(s.store, VarUnstakeSlotMax, value)
```

Comment:
Error Scenario
1. Account A get unstakes to MAX
2. decrease UnstakeSlotMax
3. How to control overflowed unstakes of Account A

Suggestion 1
- Use constant UnstakeSlotMax
- remove SetUnstakeSlotMax()

Suggestion 2
- SetUnstakeSlotMax() return error if try to decrease UnstakeSlotMax